libpod/container_internal_linux.go

```diff
@@ -1381,23 +1401,12 @@ func (c *Container) makeBindMounts() error {
 			// other container. Unless we're not creating both of
 			// them.
 			var (
-				depCtr  *Container
-				nextCtr string
+				depCtr *Container
```

Comment:
Can drop this block and make `depCtr, err = c.getRootNetNsDepCtr()` a `:=`

Will be a little cleaner

Reply:
https://github.com/containers/podman/pull/9972#discussion_r615246254